vendor/github.com/openshift/library-go/pkg/operator/staticresourcecontroller/static_resource_controller.go

```diff
@@ -0,0 +1,240 @@
+package staticresourcecontroller
```

Comment:
per discussion in office hours, @adambkaplan believes he can do some git refactoring to get this file's change minimally out of the main code change commit